### DISCUSSION TITLE
能否把翻译水平调节到专家级？

### DISCUSSION THREAD

**Discussion by ghost** [yihong0618/bilingual_book_maker#158] [Ideas]
最近看到一个 chatgpt 翻译专家版的信息，觉得要是能用到这个项目上，说不定可以提升翻译的水平。具体信息如下：
“ChatGPT 的翻譯有比 Google 翻譯更優秀嗎？”
https://medium.com/dean-lin/chatgpt-%E7%9A%84%E7%BF%BB%E8%AD%AF%E6%9C%89%E6%AF%94-google-%E7%BF%BB%E8%AD%AF%E6%9B%B4%E5%84%AA%E7%A7%80%E5%97%8E-ca876274445f

能否把这段提示放到翻译书之前，让 chatgpt 输出更好的翻译质量？

"I want you to act as an English translator, spelling corrector and improver. I will speak to you in any language and you will detect the language, translate it and answer in the corrected and improved version of my text, in English. I want you to replace my simplified A0-level words and sentences with more beautiful and elegant, upper level English words and sentences. Keep the meaning same, but make them more literary. I want you to only reply the correction, the improvements and nothing else, do not write explanations. My first sentence is “不要懷疑！我就是專家”"

**Comment by yihong0618** [comment]
@hjonline 可以，目前可以自定义 prompt 你可以用这段。

**Reply by ghost** [reply]
谢谢。我得查查如何自定义 prompt 。
